### DISCUSSION TITLE
Migration of Camunda community projects

### DISCUSSION THREAD

**Discussion by PascalLugon** [cibseven/discussions#4] [Q&A]
With the new release of CIB Seven 1.1, as expected I'm facing issues with the community projects I'm using (camunda-bpm-data and camunda-process-test-coverage). Is there a straightforward method to use them with CIB Seven?
Or are you planning to open a community section to fork some of these projects?
BR
Pascal

**Comment by OlegCIB** [comment]
Hi Pascal,

Our main goal in creating a CIB seven fork is to provide long term support for our CIB flow customers.

We would be happy if other customers could share our maintenance costs by providing them with LTS as well, but for now we are only focusing on the projects that are necessary for us and our customers.

On the other hand, as an open source project, we would of course be happy to help the community. We have not yet spoken to the maintainers of the Community Hub repositories about whether they will move to CIB seven and continue to maintain their repositories under the org.cibseven namespace.

If you are interested in contributing, we have created [cibseven-community-hub](https://github.com/cibseven-community-hub) and invited you as a maintainer, feel free to fork the projects into it!
If you need help with forking or moving to the org.cibseven namespace - let us know.
But we won't put our main focus on supporting such community repositories if our customers don't need them.

**Reply by PascalLugon** [reply]
Thx for the reply.
It's good to know that you have also a community hub.
BR
Pascal

**Reply by PascalLugon** [reply]
Hi,

I've forked and migrated 2 holunda.io projects to CIB 7 1.1.0: camunda-bpm-data and camunda-bpm-api (needed by the first one).
-> my app is working fine with CIB 7 1.1.0 and these frameworks.

Is that possible to contact you directly in order to integrate them into your community hub?

BR
Pascal

**Reply by PascalLugon** [reply]
My projects:
https://github.com/PascalLugon/camunda-bpm-data/tree/feature/CIB
https://github.com/PascalLugon/camunda-bpm-api/tree/feature/CIB

**Reply by PascalLugon** [reply]
Hello,
I've updated also some additional frameworks: camunda-platform-7-mockito, camunda-process-test-coverage, and camunda-bpm-jgiven
So my app is now fully running with CIB 7 1.1.0 and a successful run of our integration tests.
BR

**Comment by OlegCIB** [comment]
Hi @PascalLugon,

sorry it's taken us a bit longer, but we've finalized the release process for the community projects now and all of your changes have been merged into the main branches of the corresponding projects (e.g. https://github.com/cibseven-community-hub/cibseven-process-test-coverage/pull/1) and released them to [Maven Central](https://central.sonatype.com/search?q=org.cibseven.community).

Thank you very much for your contribution!

After the release, we've updated all projects to the next snapshot-versions, so that further development and/or maintenance (such as backporting commits from the camunda/holunda-repos and bumping dependency versions) can continue.

Do you want to continue maintaining the repos by creating new branches and pull requests?

Cheers,
Oleg

**Comment by PascalLugon** [comment]
Oleg,

Thanks for the feedbacks.

Since we are in an Improvement Process Sprint (Ah agility….), I’ve already started this week to migrate to the latest versions of your community libs. I was able to run our unitary tests and integration tests without any glitches.

Well done!!!

I would like to continue to participate to the maintenance. I just need to build my Github knowledge about the normal workflow (pull request…)…
@Nagra we are using Gitlab.

Could we have a short chat to discuss the next steps?

Best regards
Pascal
